showcase/src/assets/translations/nl.json

```diff
@@ -94,6 +94,15 @@
         "TITLE": "Stark Generic Search",
         "TOGGLE": "Toggle search"
       },
+      "INPUT_MASK_DIRECTIVES": {
+        "TEXT_MASK": {
+          "TITLE": "Text Mask directive",
+          "CREDIT_CARD_MASK": "Creditcardmasker",
+          "STRUCTURED_MESSAGE_MASK": "Gestructureerd communicatie berichtmasker",
```

Comment:
I think "Gestructureerde mededeling masker" is better or maybe just "Gestructureerde mededeling"